tests/testcontainers/logstash.py

```diff
@@ -63,28 +70,76 @@ def __init__(
         self._last_reset_message_count: int = 0
         self._previous_message_count: int = 0
 
+    @staticmethod
+    def _certificates_for_authority_and_server(service_identity: str, key_size: int = 1024) -> tuple[str, str]:
```

Comment:
I am fine with generating the cert in code, but what exactly are we gaining by adding this functionality in the tests?

Reply:
that in two years the tests won't broke because of the expired cert and nobody will remember where to look :D

Reply:
Fair enough lol